service/main.ml

```diff
@@ -90,10 +90,15 @@ let main config mode app capnp_address github_auth submission_uri : ('a, [`Msg o
   Lwt_main.run begin
     run_capnp capnp_address >>= fun (vat, rpc_engine_resolver) ->
     let ocluster = Option.map (Capnp_rpc_unix.Vat.import_exn vat) submission_uri in
-    let engine = Current.Engine.create ~config (Pipeline.v ?ocluster ~solver) in
+    let confs = Conf.configs in
+    let engine = Current.Engine.create ~config (Pipeline.v ?ocluster ~solver ~confs) in
     rpc_engine_resolver |> Option.iter (fun r -> Capability.resolve_ok r (Api_impl.make_ci ~engine));
     let authn = Option.map Current_github.Auth.make_login_uri github_auth in
-    let webhook_secret = Current_github.App.webhook_secret app in
+    let webhook_secret =
+      match app with
+      | Some app -> Current_github.App.webhook_secret app
+      | None -> ""
+    in
```

Comment:
This should stay as mandatory, I think the setup is there in the deployed en

Reply:
The app is not used by the pipeline yet. This is also useful for debugging purpose, we don't have to config the app stuff, to start the service.